cmd/teleirc.go

```diff
@@ -18,42 +17,43 @@ const (
 
 var (
 	flagPath    = flag.String("conf", ".env", "config file")
-	flagDebug   = flag.Bool("debug", false, "enable debugging")
+	flagDebug   = flag.Bool("debug", false, "disable debugging")
```

Comment:
I think keeping `enable debugging` is better, since this lets users know that any changes to this `Bool` adds debugging capabilities.